### ISSUE TITLE
Failed to load plugin: stickytoolbar from url http://glpi.eda-plastiques.fr/glpi/lib/tiny_mce/lib/plugins/stickytoolbar/plugin.min.js

### ISSUE THREAD

**Issue by Sismic** [InfotelGLPI/tasklists#37]
Hi, when i add a task into a status, this warnin message appears
![image](https://user-images.githubusercontent.com/6815687/91828379-5963db80-ec40-11ea-9d0c-e1c7138a4df3.png)

i can close it, all is ok

**Comment by RodolpheLP** [comment]
Hi,
Same issue there :
"_Failed to load plugin: stickytoolbar from url http://snt-glpi/glpinext/lib/tiny_mce/lib/plugins/stickytoolbar/plugin.min.js_"

version **GLPI 9.5.1** and plugin **tasklists 1.6.0**

Thanks for your help,
Rodolphe

**Comment by RodolpheLP** [comment]
> Hi,
> Same issue there :
> "_Failed to load plugin: stickytoolbar from url http://snt-glpi/glpinext/lib/tiny_mce/lib/plugins/stickytoolbar/plugin.min.js_"
> 
> version **GLPI 9.5.1** and plugin **tasklists 1.6.0**
> 
> Thanks for your help,
> Rodolphe

After a quick search I found that my GLPI repository doesn't have the necessary element.

1 - Look in "[glpi directory]/lib/tiny_mce/lib/plugins/" on my server and **stickytoolbar is not there !** (i don(t know if it is normal whit GLPI 9.5.1 ?)
2 - Search "stickytoolbar plugin" on Google
3 - So I found it : https://github.com/kurtnovack/stickytoolbar
4 - I download the last release : https://github.com/kurtnovack/stickytoolbar/releases/tag/2.2.1
5 - I uncompress it and I take only "stickytoolbar" in the "dist" directory
6 - I copy that directory in my "[glpi directory]/lib/tiny_mce/lib/plugins/" on my server

It seems that this resolves the problem.

For the moment I use this solution only in my test server (before migration GLPI from 9.4.6 to 9.5.1 in my real server).
I don't assume issue with this in production for the moment...

If this can help someone,
Regards,

Rodolphe
